autoload/neomake/makers/ft/javascript.vim

```diff
@@ -74,7 +74,7 @@ endfunction
 function! neomake#makers#ft#javascript#rjsx() abort
     return {
         \ 'exe': 'emacs',
-        \ 'args': ['%','--quick','--batch','--eval='
+        \ 'args': ['%t','--quick','--batch','--eval='
```

Comment:
@bounceme
I've assumed that the file position is relevant here (and kept the `%t`), but maybe it is not? (i.e. it can be appended?)